### PR TITLE
[#108970448] Fixed subscription.CurrentStorageAccountName display in azure cmdlets

### PR DESCRIPTION
**Pull request by hovsepm**
Addressed issue #1332

### CODE REVIEW

File: src/ServiceManagement/Profile/Commands.Profile/Microsoft.WindowsAzure.Commands.Profile.format.ps1xml [hunk 1]
```diff
@@ -76,8 +76,7 @@
                 <PropertyName>TenantId</PropertyName>
               </ListItem>
               <ListItem>
-                <Label>CurrentStorageAccountName</Label>
```
Comment (markcowl) [inline src/ServiceManagement/Profile/Commands.Profile/Microsoft.WindowsAzure.Commands.Profile.format.ps1xml]: Should call GetAccountName() here

File: src/ServiceManagement/Profile/Commands.Profile/Models/PsAzureSubscription.cs [hunk 2]
```diff
@@ -34,12 +34,10 @@ public PSAzureSubscription(AzureSubscription subscription, AzureSMProfile profil
             Accounts = profile.Accounts.Values.Where(a => a.HasSubscription(subscription.Id)).ToArray();
             IsDefault = subscription.IsPropertySet(AzureSubscription.Property.Default);
             IsCurrent = profile.Context != null && profile.Context.Subscription.Id == subscription.Id;
-            CurrentStorageAccountName = subscription.GetProperty(AzureSubscription.Property.StorageAccount);
+            CurrentStorageAccountName = GetAccountName(subscription.GetProperty(AzureSubscription.Property.StorageAccount));
```
Comment (markcowl) [inline src/ServiceManagement/Profile/Commands.Profile/Models/PsAzureSubscription.cs]: Should leave the property as is, we want to make sure current storage account is still transmitted with the full connection string

**Comment by hovsepm** [comment]
on-demand job: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/374/

**Comment by hovsepm** [comment]
new on-demand: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/375/

**Pull request merged**